lisa/sut_orchestrator/azure/common.py

```diff
@@ -151,9 +151,15 @@ class AzureVmPurchasePlanSchema:
     publisher: str
 
 
+@dataclass_json
+@dataclass
+class AzureImageSchema:
+    is_accelerated_network_supported: bool = True
```

Comment:
Let's give it more generic name, and handle more scenario. Make it the same type as `NetworkInterfaceOptionSettings.data_path`

Reply:
If so, the runbook should be defined as below if I understand right. Seems like this makes the runbook a little complex to use.
  requirement:
    azure:
       vhd: 
          network_data_path: 
            - Synthetic
            - Sriov

For user, they might not know directly if their image's network is synthetic or sriov. They can know directly if their images enable AN or support AN. 

Take marketplace image and VHD as examples:
If it is a marketplace image, we can get "**IsAcceleratedNetworkSupported**" field from image features. If it is a VHD format, in the Azure Certification scenario, the parameter passed from the upper layer is "**AcceleratedNetworking** **IsEnabled**".  So keeping the current type and name can connect with these scenarios and better understand.

Reply:
1. It has the default value to provide both. Only if the data_path is not fully supported, this field should be set.
2. In future, there may be a scenario, which support only AN. Above setting can fit more scenarios.

Reply:
Done